selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/WorkspaceShare.java

```diff
@@ -11,92 +11,118 @@
+    this.seleniumWebDriverHelper = seleniumWebDriverHelper;
     PageFactory.initElements(seleniumWebDriver, this);
   }
 
   private interface Locators {
-    String ADD_DEVELOPER_BTN = "//span[text()='Add Developer']";
-    String REMOVE_DEVELOPER_ICON =
-        "//span[text()='%s']//following::div[@tooltip='Remove member'][1]";
-    String INPUT_SHARE_DIALOG = "//md-chips[contains(@class,'share-user-input')]//input";
-    String SHARE_BTN_DIALOG = "//che-button-primary[@aria-disabled='false']//span[text()='Share']";
-    String DEVELOPER_SHARE_ITEM = "//span[text()='%s']";
-    String WARNING_DIALOG_DELETE = "//div[@class='ng-binding' and text()=\"%s\"]";
-  }
-
-  /**
-   * Adds developer to share list
-   *
-   * @param email is an email of developer into sharing list
-   */
-  public void addDeveloperToShareList(String email) {
-    new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC)
-        .until(visibilityOfElementLocated(By.xpath(Locators.ADD_DEVELOPER_BTN)))
-        .click();
-    new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC)
-        .until(visibilityOfElementLocated(By.xpath(Locators.INPUT_SHARE_DIALOG)))
-        .sendKeys(email + ",");
-    new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC)
-        .until(visibilityOfElementLocated(By.xpath(Locators.SHARE_BTN_DIALOG)))
-        .click();
-  }
-
-  /**
-   * Delete a developer from sharing list
-   *
-   * @param email is an email of developer into sharing list
-   */
-  public void deleteDeveloperFromShareList(String email) {
-    new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC)
-        .until(
-            visibilityOfElementLocated(
-                By.xpath(String.format(Locators.REMOVE_DEVELOPER_ICON, email))))
-        .click();
-  }
-
-  /** Wait the text into warning dialog delete or remove */
-  public void waitTextInWarningDialogDelete(String expText) {
-    new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC)
-        .until(
-            visibilityOfElementLocated(
-                By.xpath(String.format(Locators.WARNING_DIALOG_DELETE, expText))));
-  }
-
-  /**
-   * Wait the email of developer is present in 'Share' tab
-   *
-   * @param email the email of developer
-   */
-  public void waitDeveloperIsPresentInShareTab(String email) {
-    new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC)
-        .until(visibilityOfElementLocated(By.xpath(format(Locators.DEVELOPER_SHARE_ITEM, email))));
-  }
-
-  /**
-   * Wait the email of developer is not present in 'Share' tab
-   *
-   * @param email the email of developer
-   */
-  public void waitDeveloperIsNotPresentInShareTab(String email) {
-    new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC)
-        .until(
-            invisibilityOfElementLocated(By.xpath(format(Locators.DEVELOPER_SHARE_ITEM, email))));
+    String ADD_DEVELOPER_BTN_XPATH = "//che-button-primary[@che-button-title='Add Developer']";
```

Comment:
We don't need to shorten `BUTTON` to `BTN`, because it doesn't effectively shorten the code, but make it harder to understand.